RecoLocalTracker/SiStripZeroSuppression/python/SiStripZeroSuppression_cfi.py

```diff
@@ -28,3 +28,8 @@
                            'simSiStripDigis:ProcessedRaw',
                            'simSiStripDigis:ScopeMode' ]
 )
+
+siStripZeroSuppressionHLT = siStripZeroSuppression.clone(
+    RawDigiProducersList =cms.VInputTag(cms.InputTag("siStripDigisHLT","VirginRaw"), cms.InputTag("siStripDigisHLT","ProcessedRaw"), cms.InputTag("siStripDigisHLT","ScopeMode")
+))
```

Comment:
The modification moved here, and therefore it is here that my comment https://github.com/cms-sw/cmssw/pull/38423#discussion_r909325862 should apply now:
```suggestion
    RawDigiProducersList =[("siStripDigisHLT","VirginRaw"), ("siStripDigisHLT","ProcessedRaw"), ("siStripDigisHLT","ScopeMode")]
```